redesign-project/src/index.html

```diff
@@ -0,0 +1,67 @@
+---
+layout: layouts/default.njk
+title: Emma's Tea Spot
+---
+
+    {% include "_header.html" %}
```

Comment:
There is nothing currently inside this partial, FYI. Especially as you start to add further pages, you're going to want to move the shared header elements into the `_header.html` file.